src/docs/releases/1.9.0.md

```diff
@@ -392,4 +392,140 @@ Enhanced functionality has been implemented, giving developers the ability to co
  ```csharp
  services.Configure<EmailConfirmationTokenProviderOptions>(options => options.TokenLifespan = TimeSpan.FromDays(7));
  ```
- 
\ No newline at end of file
+ 
+ ### Settings
+
+When implementing a site settings display driver, you have the option to reload the shell (refresh the tenant). If you choose to do so, manually releasing the shell context using `await _shellHost.ReleaseShellContextAsync(_shellSettings)` is unnecessary. Instead, you should use the new `site.QueueReleaseShellContext()`. This ensures that the shell stays intact until all settings are validated. For instance, in ReverseProxySettingsDisplayDriver, the code was modified from this.
```

Comment:
I think it's enough to link to https://github.com/OrchardCMS/OrchardCore/pull/15875/files#diff-8c2b7f9a5a97fefce4c59cca868b2d2917e37138afe465fff0f08ab2ddfe6e70 (or perhaps a simpler sample, like https://github.com/OrchardCMS/OrchardCore/pull/15875/files#diff-698978f27ab80944f91418c4f4360598bee929ca432b29605d5893b053b578b5 so people can see this in context, and only the diff (here they have to eyeball the changes).

Reply:
I like to keep an example in the release doc. but what I did is I shortened it.

Reply:
As a module author, I need to know exactly what to change. Having a full example can be useful (though I think that's also more useful in its entirety), but having to process the whole thing here to understand what to change is a chore.

This is what a module author needs to do to understand what's in here (mimicking the thought process and eye movements with the highlights):


https://github.com/OrchardCMS/OrchardCore/assets/1976647/27671a23-538b-42d2-a5b0-1239b43972e7

And this is it when seeing a diff:


https://github.com/OrchardCMS/OrchardCore/assets/1976647/ed37ad0a-9ee2-425b-87b4-90fe9a629a49

It's super clear. I don't understand why we wouldn't show that you need code in a certain way with a diff that's specifically for this and every developer is familiar with.

Reply:
I understand. But we almost never reference the PR in the release document. The release docs gives a high level of the changes along with an example. That to me is enough. The release notes usually contains reference to the PR. Maybe with each important log into the release notes we would reference the PR.

Reply:
I think we can be the pioneers and be the first to link a PR from the release notes :). I just don't want our users, my colleagues included, to have to jump through hoops and give them homework when we can just tell them exactly and only what they need in a form they all know.

See for example how here we told people directly what they need? https://github.com/Lombiq/.NET-Analyzers/releases/tag/v4.0.0 Or here: https://github.com/Lombiq/UI-Testing-Toolbox/blob/dev/Lombiq.Tests.UI/Docs/Migration.md#preparing-the-web-application-project They can look up the details if they want to, but if they don't, then they have a clear "if you had this then do that" instruction. If MkDocs can have similar diff code blocks then great and let's do that, otherwise, linking to the GitHub diff is the second best option (I briefly googled and it seems that this is a feature of GitHub's Markdown and there's no equivalent in MkDocs).

Reply:
I added a reference to the PR.

Reply:
Could you please clarify why you're against just linking to a file's diff that shows what exactly the changes are in favor of keep making this section longer? This documentation section could be simultaneously shorter, and clearer.

As someone who oversees keeping half a dozen OC apps up-to-date, that also have a lot of non-OC dependencies, and work with a lot more occasionally, such release documentation matters a lot (and the lack of clarity in it costs a lot). I don't really understand why this is something we spend so much time debating. Just to make sure we're talking about the same thing: my suggestion is to add something like the below to this section, and nothing else:

```
You can see an example of how to change a driver [here](https://github.com/OrchardCMS/OrchardCore/pull/15875/files#diff-698978f27ab80944f91418c4f4360598bee929ca432b29605d5893b053b578b5).
```

Reply:
> Could you please clarify why you're against just linking to a file's diff that shows what exactly the changes are in favor of keep making this section longer? This documentation section could be simultaneously shorter, and clearer.

I am not against. I added a reference to the PR in the docs. Now any one can get to the PR and see the description or view the file diff.

Reply:
Let's leave it like that.